tools/enhancements/metadata.go

```diff
@@ -43,3 +44,79 @@ func (s *Summarizer) GetMetaData(pr int) (*MetaData, error) {
 	}
 	return NewMetaData(content)
 }
+
+// Validate returns a list of issues found with the metadata
+func (m *MetaData) Validate() []string {
+	results := []string{}
+
+	reportError := func(msg string) {
+		results = append(results, msg)
+	}
+
+	// Must have one valid tracking link and no TBD
+	foundLink := false
+	for _, trackingLink := range m.TrackingLinks {
+		if trackingLink == "TBD" {
+			reportError("'TBD' is not a valid value for tracking-link")
+			continue
+		}
+		if trackingLink == "" {
+			reportError("empty string for tracking-link")
+			continue
+		}
```

Comment:
I was trying to differentiate between a missing value and an invalid value. Do you think people would understand "could not parse tracking link ''" as meaning the value was empty and it can't be?

Reply:
Done